### ISSUE TITLE
SVG not working in Openscad or FreeCAD

### ISSUE THREAD

**Issue by jtommi** [adrianschlatter/ppf.datamatrix#19]
Hello,
I have almost no knowledge in SVG.
I want to 3D print Datamatrix barcodes.
So the plan was to generate SVGs and then use them in FreeCAD or openscad to extrude them into 3D objects.
But in FreeCAD it shows up as horizontal lines and in openscad, it looks almost like a datamatrix, but it isn't.
I opened the file in Inkscape and it works fine there.

I guess this is a limitation of how the file is generated. Can you maybe explain why? Can this be changed? etc.
Here is the genrated SVG
![letters](https://github.com/adrianschlatter/ppf.datamatrix/assets/33985391/8fbbc029-6ac6-4057-807f-ee8dd1c69877)
And here the openscad extrude
![openscad](https://github.com/adrianschlatter/ppf.datamatrix/assets/33985391/d2630558-9ac7-4149-89e0-9a66c032b9bb)

**Comment by adrianschlatter** [comment]
I cannot really follow. But the datamatrix svg really is made of lines (not areas). Maybe this is the problem?

**Comment by jtommi** [comment]
Probably.
For people that might stumble upon this trying to generate a Datamatrix as a 3D object, here is how I finally did it, thanks to your package.

I was so focused on converting/generating a datamatrix in a format that CAD software can read and use it to generate 3D objects.
Since FreeCAD allow scripting (OpenSCAD also, but not necessarily Python), I took a completely different approach, skipping the intermediary step altogether.
I used ppf.datamatrix to get a matrix of the black and white squares, like this:
`DataMatrix("hello world").matrix`
and then used this matrix to generate a square for each black dot and extrude it, completely skipping the SVG part.

Thanks a lot @adrianschlatter
